File: PYTHON/PROGRAM/Komponen.py
class Komponen:
    #konstruktor
    def __init__(self, merk = "", nama = ""):
        self.__merk = merk
        self.__nama = nama

    #getter dan setter
    def getMerk(self):
        return self.__merk
    
    def setMerk(self, merk):
        self.__merk = merk

    def getNama(self):
        return self.__nama
    
    def setNama(self, nama):
        self.__nama = nama

File: CPP/PROGRAM/Komponen.cpp
#pragma once
#include <iostream>
#include <string>

using namespace std;

class Komponen{
    //atribut
    private:
        string merk;
        string nama;

    public:
        //konstruktor
        Komponen(){
            this->merk = "";
            this->nama = "";
        }

        Komponen(string merk, string nama){
            this->merk = merk;
            this->nama = nama;
        }

        //getter dan setter
        void setMerk(string merk){
            this->merk = merk;
        }

        void setNama(string nama){
            this->nama = nama;
        }

        string getMerk(){
            return merk;
        }

        string getNama(){
            return nama;
        }

        ~Komponen(){

        }
};
